src/MOI_wrapper.jl

```diff
@@ -230,9 +235,26 @@ function MOI.supports_constraint(
     return true
 end
 
+function MOI.supports(
+    model, ::Optimizer, ::Union{MOI.ConstraintSet, MOI.ConstraintFunction},
```

Comment:
That shouldn't be defined:
```julia
julia> using MathOptInterface

julia> const MOI = MathOptInterface
MathOptInterface

julia> MOI.is_copyable(MOI.ConstraintSet())
false
```

Reply:
What's the rule for needing to `support` attributes? If `is_copyable` is `true`?

Reply:
Yes :)
https://github.com/JuliaOpt/MathOptInterface.jl/blob/0ed068ff8342cf2501305963c22708b0b18a4b80/src/attributes.jl#L157-L159